pkg/kv/kvserver/store_snapshot.go

```diff
@@ -1155,7 +1155,7 @@ func sendSnapshot(
 	// completed (such as defers that might be run after the previous message was
 	// received).
 	if unexpectedResp, err := stream.Recv(); err != io.EOF {
-		return errors.Errorf("%s: expected EOF, got resp=%v err=%v", to, unexpectedResp, err)
+		return errors.Wrapf(err, "%s: expected EOF, got resp=%v", to, unexpectedResp)
```

Comment:
I haven't looked at the possible returns of `stream.Recv()`; but this might be problematic if `err = nil` since errors.Wrapf will return nil in that case.

Reply:
Fixed